Makefile

```diff
@@ -80,7 +80,7 @@ test-acceptance-api: showfocus embed_files
 	ginkgo -nodes ${GINKGO_NODES} -stream -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/api/v1/.
 
 test-acceptance-cli: showfocus embed_files
-	ginkgo -nodes ${GINKGO_NODES} -stream -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/.
+	ginkgo -nodes ${GINKGO_NODES} -v -slowSpecThreshold 200 -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/.
```

Comment:
why not the other suites too?

Reply:
See commit d14244256dfedb53076714c66cdde9efe4e2741d